Cache-control: Adjusted description of cache directive structure (#31852)
mdn/content: files/en-us/web/http/headers/cache-control/index.md

## Syntax

> Cache directives have the following rules:

Author: ```suggestion
Cache directives follow these rules:
```

> Cache directives have the following structure:

Reviewer: We could still use "rules", or even "syntax", although your edit is already a good improvement:

```suggestion
Cache directives have the following rules:
```

Author: Yeah, that makes sense because these are still framed more as rules than as a description of the syntactic structure

Author: @bsmth I went [even further](https://github.com/mdn/content/pull/31852#discussion_r1463292651) up on re-reading it.

- Caching directives are case-insensitive. However, lowercase is recommended because some implementations do not recognize uppercase directives.
- Multiple directives are permitted and must be comma-separated (e.g., `Cache-control: max-age=180, public`).

> - Some directives have an optional argument. When an argument is supplied, it is separated from the directive name by a `=`. Typically arguments for the directives are integers and so are not quoted strings, just single integers (e.g. `Cache-control: max-age=12`).

Reviewer: Minor suggestion, what do you think?

```suggestion
- Some directives have an optional argument. When an argument is provided, it is separated from the directive name by an equals symbol (`=`). Typically, arguments for the directives are integers and are therefore not enclosed in quote characters (e.g., `Cache-control: max-age=12`).
```

Author: Yup, that's even clearer! "Quoted string" is definitely jargon that I managed to use while trying to avoid jargon 🤣
